src/Services/membersService.js

```diff
@@ -1,10 +1,8 @@
 import axios from 'axios';
 
-const MEMBERS_URL = 'http://ongapi.alkemy.org/api/members';
```

Comment:
instead of delete the MEMBER_URL, import the (process.env.REACT_APP_MEMBERS_URL) here